tsdapiclient/guide.py

```diff
@@ -163,12 +163,12 @@
 Import links can be used to upload data to a specific project. To upload a file
 with a password-protected import link, do the following:
     
-    tacl --api-key @<path-to-api-key-file> --link-id <link_id> --secret-challenge-file @<path-to-secret-file> --upload myfile
```

Comment:
I believe the <> markers to indicate that the content should be replaced should still be included, like the other arguments there.  Should also be used in &lt;myfile>.